From fdca02abf6bd87f747b1d52f738d9c3a0a965ca7 Mon Sep 17 00:00:00 2001
From: Benoy Antony <benoy@apache.org>
Date: Sat, 30 Aug 2014 12:49:19 -0700
Subject: [PATCH 245/791] HADOOP-10833. Remove unused cache in UserProvider.
 (Benoy Antony)

(cherry picked from commit 258c7d0f53fbdf4b0b9cae901701176e3e70c4fe)

Conflicts:
	hadoop-common-project/hadoop-common/CHANGES.txt
---
 .../apache/hadoop/security/alias/UserProvider.java |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/alias/UserProvider.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/alias/UserProvider.java
index 9e724c0..e22c958 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/alias/UserProvider.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/alias/UserProvider.java
@@ -21,9 +21,7 @@
 import java.io.IOException;
 import java.net.URI;
 import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 
 import org.apache.hadoop.classification.InterfaceAudience;
 import org.apache.hadoop.conf.Configuration;
@@ -41,8 +39,6 @@
   public static final String SCHEME_NAME = "user";
   private final UserGroupInformation user;
   private final Credentials credentials;
-  private final Map<String, CredentialEntry> cache = new HashMap<String, 
-      CredentialEntry>();
 
   private UserProvider() throws IOException {
     user = UserGroupInformation.getCurrentUser();
@@ -86,7 +82,6 @@ public void deleteCredentialEntry(String name) throws IOException {
       throw new IOException("Credential " + name + 
           " does not exist in " + this);
     }
-    cache.remove(name);
   }
 
   @Override
-- 
1.7.9.5

